#The latest Intel compiler (2016,2017) complains about the incompatibility of "void *" argument
#being incompatible with parameter of type "int *"
#M. Fujinaga (fujinaga@ualberta.ca)
--- processor_support.h.orig	2017-02-02 21:48:42.436432756 +0000
+++ processor_support.h	2017-02-02 21:48:54.419309743 +0000
@@ -44,8 +44,8 @@
 
     try {
 #if ( defined(USING_INTEL_COMPILER) || defined(USING_MSC_COMPILER) )
-        __cpuid((void *) &regs,0); // test if __cpuid() works, if not catch the exception
-        __cpuid((void *) &regs,0x1); // POPCNT bit is bit 23 in ECX
+        __cpuid((int *) &regs,0); // test if __cpuid() works, if not catch the exception
+        __cpuid((int *) &regs,0x1); // POPCNT bit is bit 23 in ECX
 #elif defined(USING_GCC_COMPILER)
         __get_cpuid(0x1, &regs.EAX, &regs.EBX, &regs.ECX, &regs.EDX);
 #else
